src/khoj/main.py

```diff
@@ -160,7 +160,9 @@ def start_server(app, host=None, port=None, socket=None):
     if socket:
         uvicorn.run(app, proxy_headers=True, uds=socket, log_level="debug", use_colors=True, log_config=None)
     else:
-        uvicorn.run(app, host=host, port=port, log_level="debug", use_colors=True, log_config=None)
+        uvicorn.run(
+            app, host=host, port=port, log_level="debug", use_colors=True, log_config=None, timeout_keep_alive=60
```

Comment:
What does the timeout_keep_alive do here? Is this required?

Reply:
It allows clients to maintain more long-living sessions with the server. This can help minimize the overhead of establishing a connection to the backend, therefore providing faster responses. This specific code only affects people self-hosting locally.

Reply:
It allows clients to maintain more long-living sessions with the server. This can help minimize the overhead of establishing a connection to the backend, therefore providing faster responses. This specific code only affects people self-hosting locally.